jetson/odrive_bridge/src/__main__.py

```diff
@@ -287,16 +287,16 @@ def get_state(self):
 """
 
 
-def publish_state_msg(msg, state):
-    global legal_controller
-    msg.state = states.index(state)
-    msg.controller = legal_controller
+def publish_state_msg(msg, state_string):

```

Comment:
this looks a bit convoluted can you add a comment above for clarity

Reply:
yea sure